### PR TITLE
Added website route to adikeshri

### PR DESCRIPTION
**Pull request by adikeshri**

### CODE REVIEW

**Comment by dntandan** [comment]
Solve issues #50

File: components/Team.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function Team() {
                 img="https://picsum.photos/350"
                 designation="Full Stack Developer"
                 github="github-link"
-                web="web-link"
+                web="https://adikeshri.ml"
```
Comment (dntandan) [inline components/Team.tsx:22]: is the url correct?

**Review by dntandan — COMMENTED** [review]
do this part is correct?

File: components/Team.tsx (hunk 1 above)
Comment (dntandan) [reply components/Team.tsx:22]: @adikeshri

Comment (adikeshri) [reply components/Team.tsx:22]: Yes

**Pull request merged**